### PR TITLE
Add LCD support for distribution module

### PR DESCRIPTION
**Pull request by willclarktech**
Part of #276

### CODE REVIEW

File: packages/sdk38/src/lcdapi/distribution.spec.ts [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    gas: "1500000", // 1.5 million
+  };
+
+  beforeAll(async () => {
+    if (wasmdEnabled()) {
+      const wallet = await Secp256k1Wallet.fromMnemonic(faucet.mnemonic);
+      const client = new SigningCosmosClient(wasmd.endpoint, faucet.address, wallet, {});
+
+      const chainId = await client.getChainId();
+      const msg = {
```
Comment (webmaster128) [inline packages/sdk38/src/lcdapi/distribution.spec.ts:38]: For this one we even have a type: `const msg: MsgDelegate = {`

File: packages/sdk38/src/lcdapi/distribution.spec.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  describe("validator", () => {
+    it("works", async () => {
+      pendingWithoutWasmd();
+      const client = makeDistributionClient(wasmd.endpoint);
+      const response = await client.distribution.validator(validatorAddress);
+      expect(response).toEqual({
+        height: jasmine.stringMatching(nonNegativeIntegerMatcher),
+        result: {
+          // TODO: This smells like a bug in the backend to me
+          operator_address: Bech32.encode("cosmos", Bech32.decode(validatorAddress).data),
```
Comment (webmaster128) [inline packages/sdk38/src/lcdapi/distribution.spec.ts:119]: @ethanfrey? 👆

**Comment by webmaster128** [comment]
@willclarktech can you update this (and merge when green)?

File: packages/sdk38/src/lcdapi/distribution.spec.ts (hunk 2 above)
Comment (ethanfrey) [reply packages/sdk38/src/lcdapi/distribution.spec.ts:118]: Yes this does seem like they use the wrong bech32 prefix here.

**Pull request merged**